### PR TITLE
adding a prefix to introspection so multiple can be running

### PR DESCRIPTION
**Pull request by LauraStelzner-TRI**
Added name to TreeStatus so updating fill in node names.  Adding and optional prefix to Introspection so that you can have multiple running to track different parts of the tree

### CODE REVIEW

File: src/task_behavior_ros/introspection.py [hunk 1]
```diff
@@ -178,7 +179,7 @@ def _force_node(self, msg):
         for i, node_id in enumerate(msg.id):
             if node_id in self.node_map.keys():
                 rospy.loginfo(
-                    "forcing "+str(node_id)+" to " + str(msg.status[i].status))
+                    "forcing " + str(node_id) + " to " + str(msg.status[i].status))
```
Comment (tinawright) [inline src/task_behavior_ros/introspection.py:182]: generally this is more efficient:
rospy.loginfo("forcing %s to %s", node_id, msg.status[i].status)

**Review by tinawright — APPROVED** [review]

**Review by athackst — COMMENTED** [review]
I'm not quite sure how you intend on using prefix?  It doesn't look like anything here is changing it from the default or unit testing multiples?

**Comment by LauraStelzner-TRI** [comment]
This was my first attempt on how we could track failures of specific tasks. This is so you could introspect a sub branch and the whole tree.  The whole tree would use the default so that tree_viewer would still work.  Then you could have a specific branch you are interested in that you want to get regular status publish on a different set of topics. Since creating this PR I've changed my approach, so we could decide to delete these changes.

**Comment by LauraStelzner-TRI** [comment]
Changing my approach, closing PR.

**Pull request closed without merging**